client/blocks/edit-gravatar/test/index.jsx

```diff
@@ -37,172 +78,147 @@ describe( 'EditGravatar', () => {
 
 	describe( 'component rendering', () => {
 		test( 'displays a Gravatar', () => {
-			const wrapper = shallow( <EditGravatar translate={ noop } user={ user } /> );
-			expect( wrapper.find( Gravatar ).length ).toEqual( 1 );
+			render( <EditGravatar { ...props } /> );
+			expect( screen.queryByTestId( 'gravatar' ) ).toBeVisible();
 		} );
 
 		test( 'contains a file picker that accepts images', () => {
-			const wrapper = shallow( <EditGravatar translate={ noop } user={ user } /> );
-			const filePicker = wrapper.find( FilePicker );
-			expect( filePicker.length ).toEqual( 1 );
-			expect( filePicker.prop( 'accept' ) ).toEqual( 'image/*' );
+			render( <EditGravatar { ...props } /> );
+			expect( screen.queryByTestId( 'file-picker' ) ).toBeVisible();
+			expect( FilePicker ).toHaveBeenCalledWith(
+				expect.objectContaining( { accept: 'image/*' } ),
+				expect.anything()
+			);
 		} );
 
 		test( 'does not display the image editor by default', () => {
-			const wrapper = shallow( <EditGravatar translate={ noop } user={ user } /> );
-			expect( wrapper.find( ImageEditor ).length ).toEqual( 0 );
+			render( <EditGravatar { ...props } /> );
+			expect( screen.queryByTestId( 'image-editor' ) ).not.toBeInTheDocument();
 		} );
 
 		test( 'indicates when Gravatar is hidden', () => {
-			const wrapper = shallow(
-				<EditGravatar translate={ noop } user={ user } isGravatarProfileHidden={ true } />
-			);
-			expect( wrapper.find( '.edit-gravatar .edit-gravatar__gravatar-is-hidden' ) ).toHaveLength(
-				1
-			);
+			render( <EditGravatar { ...props } isGravatarProfileHidden={ true } /> );
```

Comment:
Nit: I realize this was like that before, but we can omit `={ true }` for boolean props.

```suggestion
			render( <EditGravatar { ...props } isGravatarProfileHidden /> );
```

Reply:
Fixed with https://github.com/Automattic/wp-calypso/pull/66808/commits/06eb08d1d25ec79624ff537326866c554dac6628